JiraPS/Internal/ConvertTo-JiraServerInfo.ps1

```diff
@@ -0,0 +1,47 @@
+function ConvertTo-JiraServerInfo {
+    [CmdletBinding()]
+    param(
+        [Parameter(
+            Position = 0,
+            Mandatory = $false,
+            ValueFromPipeline = $true
+        )]
+        [PSObject[]] $InputObject
+    )
+
+    process {
+        foreach ($i in $InputObject) {
+            # Write-Debug "Processing object: '$i'"
+
+            # Write-Debug "Defining standard properties"
+            $props = @{
+                'BaseURL'        = $i.baseUrl
+                'Version'        = [Version]($i.version)
```

Comment:
My JIRA Cloud instance is throwing a fun problem. Seen this yet?

> Cannot convert value "1001.0.0-SNAPSHOT" to type "System.Version". Error: "Input string was not in a 
correct format."

Reply:
Nope... Didn't know that the version could contain text... Sh*t